File: dart/com/spoonacular/client/model/inline_response20054.dart
part of openapi.api;

class InlineResponse20054 {
  
  List<InlineResponse20054Videos> videos = [];
  
  int totalResults = null;
  InlineResponse20054();

  @override
  String toString() {
    return 'InlineResponse20054[videos=$videos, totalResults=$totalResults, ]';
  }

  InlineResponse20054.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['videos'] == null) {
      videos = null;
    } else {
      videos = InlineResponse20054Videos.listFromJson(json['videos']);
    }
    if (json['totalResults'] == null) {
      totalResults = null;
    } else {
          totalResults = json['totalResults'];
    }
  }

  Map<String, dynamic> toJson() {
    Map <String, dynamic> json = {};
    if (videos != null)
      json['videos'] = videos;
    if (totalResults != null)
      json['totalResults'] = totalResults;
    return json;
  }

  static List<InlineResponse20054> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20054>() : json.map((value) => new InlineResponse20054.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20054> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineResponse20054>();
    if (json != null && json.isNotEmpty) {
      json.forEach((String key, dynamic value) => map[key] = new InlineResponse20054.fromJson(value));
    }
    return map;
  }
}

File: csharp/src/main/CsharpDotNet2/spoonacular/Model/InlineResponse20054.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace spoonacular.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20054 {
    /// <summary>
    /// Gets or Sets Videos
    /// </summary>
    [DataMember(Name="videos", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "videos")]
    public List<InlineResponse20054Videos> Videos { get; set; }

    /// <summary>
    /// Gets or Sets TotalResults
    /// </summary>
    [DataMember(Name="totalResults", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "totalResults")]
    public int? TotalResults { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20054 {\n");
      sb.Append("  Videos: ").Append(Videos).Append("\n");
      sb.Append("  TotalResults: ").Append(TotalResults).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
